ND Supporting Document 2_2.adoc

```diff
@@ -2833,8 +2733,84 @@ Test 1b: The evaluator shall then 'break' the chain used in Test 1a by either re
++
+If the time period selection in FIA_AFL.1.2 is included in the ST, then the evaluator shall wait for just less than the time period configured in Test 1 and show that an authorisation attempt using valid credentials does not result in successful access. The evaluator shall then wait until just after the time period configured in Test 1 and show that an authorisation attempt using valid credentials results in successful access.
+
+==== FIA_PMG_EXT.1 Password Management
+
+===== TSS
+
+[arabic, start=150]
+. The evaluator shall examine the TSS to determine that it contains the lists of the supported special character(s) and minimum and maximum number of charters supported for administrator passwords.
+
+===== Guidance Documentation
+
+[arabic, start=151]
+. The evaluator shall examine the guidance documentation to determine that it:
+[loweralpha]
+.. identifies the characters that may be used in passwords and provides guidance to security administrators on the composition of strong passwords, and
+.. provides instructions on setting the minimum password length and describes the valid minimum password lengths supported.
+
+===== Tests
+
+[arabic, start=152]
+. The evaluator shall perform the following tests.
+[loweralpha]
+.. Test 1: The evaluator shall compose passwords that meet the requirements in some way. For each password, the evaluator shall verify that the TOE supports the password. While the evaluator is not required (nor is it feasible) to test all possible compositions of passwords, the evaluator shall ensure that all characters, and a minimum length listed in the requirement are supported and justify the subset of those characters chosen for testing.
+.. Test 2: The evaluator shall compose passwords that do not meet the requirements in some way. For each password, the evaluator shall verify that the TOE does not support the password. While the evaluator is not required (nor is it feasible) to test all possible compositions of passwords, the evaluator shall ensure that the TOE enforces the allowed characters and the minimum length listed in the requirement and justify the subset of those characters chosen for testing.
+
+==== FIA_UAU.7 Protected Authentication Feedback
+
+===== TSS
+
+[arabic, start=160]
+. None.
+
+===== Guidance Documentation
+
+[arabic, start=161]
+. The evaluator shall examine the guidance documentation to determine that any necessary preparatory steps to ensure authentication data is not revealed while entering for each local login allowed.
+
+===== Tests
+
+[arabic, start=162]
+. The evaluator shall perform the following test for each method of local login allowed:
+[loweralpha]
+.. Test 1: The evaluator shall locally authenticate to the TOE. While making this attempt, the evaluator shall verify that at most obscured feedback is provided while entering the authentication information.
```

Comment:
Test 1: The evaluator shall locally authenticate to the TOE if available,

Reply:
FIA_UAU.7 was moved to Appendix B, which contains selection-based SFRs.  An update to Test 1 shouldn't be necessary.